### ISSUE TITLE
Failed to load widgets/battery/icons/battery--nan.svg

### ISSUE THREAD

**Issue by Alechan** [PapyElGringo/material-awesome#30]
The error is printed on console when testing with Xephyr:
```
$ Xephyr :1 -ac -br -noreset -screen 1152x720
$ DISPLAY=:1.0 awesome -c ~/.config/awesome/rc.lua

```
I assume that it's because I'm on desktop (so no battery).

When adding that file (for example, `cp battery-unknown.svg battery--nan.svg`) the error disappears

**Comment by PapyElGringo** [comment]
Yeah I need to make the widget list depending of the hardware capabilities.
